test/common/common/abseil_optional_test.cc

```diff
@@ -0,0 +1,18 @@
+#include "absl/types/optional.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+TEST(abseil_optional , basic_operations) {
```

Comment:
You can skip this I think, Abseil should have its own set of tests (plus, most of this is a pretty big violation of Envoy C++ style ;) )

Reply:
Sure, added this for my own checks.

Reply:
For your reference, here are the absl::optional<> tests: https://github.com/abseil/abseil-cpp/blob/master/absl/types/optional_test.cc

Reply:
@ccaraman, Thanks!